package-lock.json

```diff
@@ -50,7 +50,7 @@
         "prettier": "2.4.1",
         "request": "2.85.0",
         "semver": "5.5.0",
-        "sonarqube-scanner": "2.8.1",
+        "sonarqube-scanner": "3.0.0",
```

Comment:
Was all the deletion of this package-lock the result of upgrading the scanner here ?

Reply:
I don't have any intentions to upgrade packages and i did not add a new one, no idea why this file was updated. I reverted my changes here!